src/lib.rs

```diff
@@ -35,8 +35,8 @@
 //!     .body("This has nothing to do with emails.\nIt should not go away until you acknowledge it.")
 //!     .icon("thunderbird")
 //!     .appname("thunderbird")
-//!     .hint(Hint::Category("email".to_owned()))
-//!     .hint(Hint::Resident(true)) // this is not supported by all implementations
+//!     // .hint(Hint::Category("email".to_owned()))
+//!     // .hint(Hint::Resident(true)) // this is not supported by all implementations
```

Comment:
left over?

Reply:
the problem here is that `cargo test` fails on windows because the docs build fails: the "hint" method doesn't exist. not sure how to fix it.

Reply:
does a `#[cfg(...)]` work here?